### PR TITLE
Implement check for adjacency between 2 given nodes

### PR DESCRIPTION
**Pull request by lhckb**

### CODE REVIEW

File: src/components/GraphEditor/Editors/MoreInfosEditor/CheckAdjacency/CheckAdjacency.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      const nodeLabel = cyRef.getElementById(nodeId).data('label');
+
+      return nodeLabel;
+    });
+
+    return (
+      <>
+        <Container>
+          {sourceNode === '' && <InstructionBox content={'Click source node'} />}
+          {sourceNode !== '' && targetNode === '' && <InstructionBox content={'Click target node to check adjacency'} />}
```
Comment (EduardoAngelo2803) [inline src/components/GraphEditor/Editors/MoreInfosEditor/CheckAdjacency/CheckAdjacency.tsx]: Click target node acredito que fica melhor pra manter o padrão de addNode e DijkstraAlgorithm

**Review by EduardoAngelo2803 — COMMENTED** [review]
'Click target node' acredito que fica melhor pra manter o padrão de addNode e DijkstraAlgorithm

**Review by EduardoAngelo2803 — APPROVED** [review]
LGTM

File: src/components/GraphEditor/Editors/MoreInfosEditor/CheckAdjacency/CheckAdjacency.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      const nodeLabel = cyRef.getElementById(nodeId).data('label');
+
+      return nodeLabel;
+    });
+
+    return (
+      <>
+        <Container>
+          {sourceNode === '' && <InstructionBox content={'Click source node'} />}
+          {sourceNode !== '' && targetNode === '' && <InstructionBox content={'Click target node'} />}
```
Comment (vitorvelozo) [inline src/components/GraphEditor/Editors/MoreInfosEditor/CheckAdjacency/CheckAdjacency.tsx]: 61 and 62 should be above Container
Currently margin-top: 30px is being applied to the InstructionBoxes, causing them to go lower than expected

**Review by vitorvelozo — CHANGES_REQUESTED** [review]

**Review by vitorvelozo — APPROVED** [review]
LGTM

**Pull request merged**